### PR TITLE
Compatibility with Rails 4.x

### PR DESCRIPTION
**Pull request by pschambacher**
cc @steved555 @grosser 

I've dropped Appraisal, Rails 2.x and 3.0 but if we really want to keep it we can just update these version checks of Rails version.

### CODE REVIEW

**Comment by pschambacher** [comment]
Also, major or minor bump?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 script: "bundle exec rake test"
 bundler_args: ""
 rvm:
-  - ree
   - 1.9.3
   - 2.0.0
```
Comment (grosser) [inline .travis.yml]: 2.1 to be safe

File: Gemfile [hunk 2]
```diff
@@ -5,7 +5,6 @@ gem "rake"
 gem "mocha"
 gem "minitest"
 gem "minitest-rg"
```
Comment (grosser) [inline Gemfile]: If you move this into gemspec as .add_development_dependency all the gemfiles get dry :)

**Comment by grosser** [comment]
minor bump is good enough I guess ... won't get installed by accident since it has a dependency on activerecord

**Comment by grosser** [comment]
:+1:

**Comment by pschambacher** [comment]
@grosser @staugaard @morten if one of you guys can rake release the gem that would be fantastic :smile:

**Comment by grosser** [comment]
1.2.0 is out!

**Comment by pschambacher** [comment]
Thanks Michael

**Pull request merged**